src/V3LinkJump.cpp

```diff
@@ -128,6 +128,19 @@ class LinkJumpVisitor final : public VNVisitor {
             return labelp;
         }
     }
+    void addPrefixToBlocksRecurse(AstNode* nodep) {
+        // Add do_while_ prefix to blocks
+        // Used to not have blocks with duplicated names
+        if (AstBegin* const beginp = VN_CAST(nodep, Begin)) {
+            if (beginp->name() != "") beginp->name("do_while_" + beginp->name());
```

Comment:
```suggestion
            if (beginp->name() != "") beginp->name("__Vdo_while_" + beginp->name());
```

To avoid an (unlikely) conflict with a user's "do_while_0".